packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -76,7 +76,7 @@ async function waitForEvent(
 }
 
 describe.each([
-  ["internal", undefined],
+  ["lucene", undefined],
```

Comment:
Funny bug that must have existed for a while, but when this was set to "internal" it was skipping both SQS and Lucene tests. Whoops.